src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorHelpers.cs

```diff
@@ -394,67 +410,120 @@ int sgn(T val)
 
 			bgraCheckeredPixelData.Capacity = width * height * 4;
 
-			//WorkItemHandler workItemHandler =
-			//(IAsyncAction workItem) =>
-			await Task.Run(() =>
-			{
-				for (int y = 0; y < height; y++)
+			WorkItemHandler workItemHandler =
```

Comment:
I intentionally ported this portion to the modern async/await pattern in C#. UWP/WinUI in C++ obviously doesn't have that pattern available.

I suppose if Uno now fully supports what was done in UWP here the code is ok. But then again I wonder if this is one of the places it is better to do what C# does considering the code is now C# instead of C++.